homeassistant/components/http/__init__.py

```diff
@@ -362,9 +362,11 @@ def json(self, result, status_code=200, headers=None):
         """Return a JSON response."""
         msg = json.dumps(
             result, sort_keys=True, cls=rem.JSONEncoder).encode('UTF-8')
-        return web.Response(
+        j = web.Response(
```

Comment:
**Preference:**
Please name this `response` instead of `j`.

Reply:
@elupus FYI: you renamed it to `request` instead of `response`, as requested.